cypress/integration/geotag-dialog-add-location.js

```diff
@@ -10,27 +10,30 @@ dragging the marker is NOT covered yet, esp given the weirdness with map coordin
 and the fact that we are technically already govering reverse geocode coverage,
 but it might be good to revisit, say, if we start using cypress to test other drag n drop functionality in the app
 */
-describe('Location can be confirmed by clicking on the map and confirming location', () => {
-  it('Opens the GeotagDialog & confirms or clears location via the popup button', () => {
-    cy.visit('/', { timeout: 10000 })
-    // need to dismiss the welcome message first
-    cy.get('.welcome-panel > .close').click()
-    cy.get('.street-metadata-map').click()
-    cy.get('.geotag-dialog').click()
-    cy.get('.geotag-location-label').should('have.text', 'Dabola, Guinea')
-    // something goofy going on with the way these coords relate to the map
-    cy.get('#map').click(625, 300)
-    // without waiting, cypress has trouble finding the updated location
-    cy.wait(100)
-    cy.get('.geotag-location-button').click({ force: true })
-    // this is currently the result when you click in center of the map at current testing viewport size
-    // there might be a less brittle way to test this?
-    // for some reason, the inital result returned from reverse geocoding these coords is Igueben, Nigeria
-    // but switches to Edo when confirming location. this might be some buggy behavior with how we use Pelias in this context
-    cy.get('span.street-metadata-map > a').should('have.text', 'Edo, Nigeria')
+context(
+  'Location can be confirmed by clicking on the map and confirming location',
+  () => {
+    before(() => {
+      cy.server()
+      cy.route(
+        ' https://api.geocode.earth/v1/reverse*',
```

Comment:
@whitmanschorn am i missing something? I would expect us to be using the pelias endpoint to test reverse geocoding? Or is it not relevant since we're mocking the result?